src/ServiceControl/Notifications/Email/CustomChecksMailNotification.cs

```diff
@@ -0,0 +1,80 @@
+﻿namespace ServiceControl.Notifications.Email
+{
+    using System;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Contracts.CustomChecks;
+    using Infrastructure.DomainEvents;
+    using NServiceBus;
+    using ServiceBus.Management.Infrastructure.Settings;
+
+    class CustomChecksMailNotification : IDomainHandler<CustomCheckFailed>, IDomainHandler<CustomCheckSucceeded>
+    {
+        readonly IMessageSession messageSession;
+        readonly string instanceName;
+        string instanceAddress;
+        string[] serviceControlHealthCustomCheckIds = {
```

Comment:
These are good points. What I was planning to do is to have a dedicated PR that aligns Custom Check Categories and does the filtering based on that value. I'll make sure that we do escape for the separator in the configuration so that there are no disallowed values.

That said I propose to have that in a dedicated PR.